.github/workflows/ci.yml

```diff
@@ -59,9 +94,11 @@ jobs:
           flag-name: run-${{ matrix.node-version }}
           parallel: true
       - name: Run spec tests
-        run: npx lerna run spec --concurrency 1
-      - name: Run integration tests
-        run: npx lerna run integration --concurrency 1
+        run: cd engines/query-sparql && yarn run spec
+      - run: cd engines/query-sparql && yarn run integration
+      - run: cd engines/query-sparql-file && yarn run integration
+      - run: cd engines/query-sparql-rdfjs && yarn run integration
+      - run: cd engines/query-sparql-rdfjs-lite && yarn run integration
```

Comment:
I've done this due to there seeming to be an issue with the way `lerna run` plays with the `windows-latest` CI https://github.com/lerna/lerna/issues/4014.

Reply:
Could you add a TODO comment in the workflow file referring to the lerna issue together with the old `lerna run` command?